pkg/collectors/customresourcedefinitions.go

```diff
@@ -0,0 +1,163 @@
+/*
```

Comment:
All the other files are called by the singular name of the object. What are your thoughts?

Reply:
Agreed, let's be consistent.